### PR TITLE
Update plot_estimate_conditional_quantile.py

### PR DESCRIPTION
**Pull request by adutfoy**
The method computeConditionalQuantile computes the quantile of the last component given all the other variables: if X=(X_1, ..., X__d), then computeQuantile computes the quantile of X_d given that (X_1, .., X_{d-1})=(x_1, ..., x_{d-1}). The example was illustrating the quantile of X_1 given X_2.

### CODE REVIEW

File: python/doc/examples/data_analysis/distribution_fitting/plot_estimate_conditional_quantile.py [hunk 1]
```diff
@@ -7,9 +7,9 @@
 # From a multivariate data sample we estimate its distribution with kernel smoothing. Here we present a bivariate distribution  X= (X1, X2). We use the `computeConditionalQuantile` method to estimate the 90% quantile :math:`Q_1` of the conditional variable :math:`X1|X2` :
```
Comment (jschueller) [inline python/doc/examples/data_analysis/distribution_fitting/plot_estimate_conditional_quantile.py:7]: X2|X1 here too then (at end of line) ?

File: python/doc/examples/data_analysis/distribution_fitting/plot_estimate_conditional_quantile.py [hunk 2]
```diff
@@ -191,7 +191,7 @@
                  '$x_2$', '$Q_1(x_2)$', True, '')
```
Comment (josephmure) [inline python/doc/examples/data_analysis/distribution_fitting/plot_estimate_conditional_quantile.py:191]: I think the title of the last graph is also wrong: it should be X2 | X1=x1

File: python/doc/examples/data_analysis/distribution_fitting/plot_estimate_conditional_quantile.py (hunk 1 above)
Comment (adutfoy) [reply python/doc/examples/data_analysis/distribution_fitting/plot_estimate_conditional_quantile.py]: Yes... it has been included in the proposition of Joseph.

File: python/doc/examples/data_analysis/distribution_fitting/plot_estimate_conditional_quantile.py (hunk 2 above)
Comment (adutfoy) [reply python/doc/examples/data_analysis/distribution_fitting/plot_estimate_conditional_quantile.py]: Ok. I change it.

**Review by josephmure — APPROVED** [review]

**Comment by jschueller** [comment]
moved to #2112

**Pull request closed without merging**